src/osdc/Objecter.cc

```diff
@@ -4235,7 +4235,7 @@ void Objecter::_finish_pool_stat_op(PoolStatOp *op, int r)
   delete op;
 }
 
-void Objecter::get_fs_stats(ceph_statfs& result,
+void Objecter::get_fs_stats(struct ceph_statfs& result,
```

Comment:
what compiling error are you running into? i am able to "make osdc" just fine locally.

Reply:
@hjwsm1989 Agree with kefu.
Just do recent clone. build it. `make osdc` complies fine without your change.

Reply:
@tchaikov @amitkumar50 I don't remember what error now, so just close it.